lebedigital/calibration/forwardmodel_linear_elastic_cylinder.py

```diff
@@ -50,21 +49,26 @@ def response(self, inp: dict) -> dict:
         parameters['log_level'] = 'WARNING'
         parameters['bc_setting'] = 'fixed'
         parameters['dim'] = 3
+
+        # as we know this problem is linear elastic, there is no point in solving it multiple times
+        # a test load is applied and then interpolated to the load list
+        test_load = -0.05
```

Comment:
Could we somehow store this result in an intermediate yaml file such that for same inputs/outputs we could store the slope (with reading in the file instead of solving might be much faster)

Reply:
This approach currently only works as we deal with a linear material model, so storing this would not be a general solution/method.